packages/dev/serializers/src/glTF/2.0/glTFExporter.ts

```diff
@@ -52,7 +52,7 @@ import { MultiMaterial } from "core/Materials/multiMaterial";
 // Matrix that converts handedness on the X-axis.
 const convertHandednessMatrix = Matrix.Compose(new Vector3(-1, 1, 1), Quaternion.Identity(), Vector3.Zero());
 
-function isNoopNode(node: Node, useRightHandedSystem: boolean): boolean {
+export function isNoopNode(node: Node, useRightHandedSystem: boolean): boolean {

```

Comment:
We should not do this as it forces usdSerializer to have a dep on gltfExporter. The function could be move in a shared util I guess ?

Reply:
You mean, adding this function somewhere in the Babylon.js core package? I don't think we want to create a new package just for this?

Reply:
We could just go back to the original check that just looks at the scaling and inverts it?

I just added this per request, so Id need some direction on what to do if we are removing it.

Reply:
you should not remove it but move it into a shared file probably in core. I d say a sceneSerializerTools under misc ?

Reply:
As this is all a party of the serializers package, we can simply have a shared directory in this package without involving changes in core, no?

Reply:
Oh yup clearly a better idea !!!

Reply:
i believe this can be resolved?